CMSgov/mint-app
# [EASI-2500] - Enum and text translation updates

# EASI-2500

## Changes and Description

- Added enum for `APPLICATION_COLLECTION_TOOL` and text translation - (Participants and Providers page 2)
- LOI text updates - (Participants and Providers page 2)
- Learning strategy text updates - (Op Eval and Learning page 5)
- Fixed type in BE and FE for `PlanContactUpdated` and `PlanContactUpdatedNote` (General Characteristics page 2)

## How to test this change

- Verify UI updates for each text revision
- Verify agains Figma links listed in JIRA ticket

## Notes

- The order of the new enum in the radio list doesn't quite match the order Figma has but that's okay

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
